src/components/ui/change-class.tsx

```
@@ -1,42 +1,51 @@
     setIsNavigatingToSchedule(true);
-    router.push('/schedule');
-  };
-
-  const handleBuyNewPackage = () => {
-    setIsNavigatingToPackages(true);
-    const params = new URLSearchParams();
-    params.append('consumerId', consumerId);
-    if (classId) params.append('classId', classId);
-    if (now) params.append('now', now);
-    router.push(`/buy-packages?${params.toString()}`);
+    router.push('/schedule?consumerId=' + consumerId);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)